alex/applications/PublicTransportInfoEN/data/expand_boroughs_script.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
```

Comment:
What is this for? In Czech, we need the expansions due to inflection. But there's no inflection in English...